convmelspec/mil.py

```diff
@@ -1,99 +0,0 @@
-# Copyright 2022 Adobe. All rights reserved.
-# This file is licensed to you under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License. You may obtain a copy
-# of the License at http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Instead of deleting this, can we keep as a historical example? I think this can be useful as an example for ISTFT, following the request in the CoreML PR.